lib/cucumber/cli/options.rb

```diff
@@ -464,6 +465,31 @@ def extract_environment_variables
         end
       end
 
+      def adjust_line_numbers
+        @args.map! do |arg|
+          next arg unless arg =~ /:\d+$/
+          line_numbers = arg.split(':')
+          file = line_numbers.shift
+          lines = File.readlines(file)
+          line_numbers.map! do |line_number|
+            line_number.to_i.downto(1).find do |i|
+              line = lines[i - 1]
+              raise ScenarioOutlineFound if line =~ /^\s+Scenario Outline:/
+              line =~ /^\s+Scenario:/
```

Comment:
A) This does not handle feature files in other language than English.
B) This does not handle all alternatives possible in a feature file in English (in Gherkin today "Example" is an alternative to "Scenario")